tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -322,7 +332,8 @@ void _printUsage(ArgParser parser) {
   print('');
   print('${Colorize('COMMANDS')..bold()}');
   print('');
-
+  _printCommandUsage(
+      cmdVersion, 'Prints the current active version of serverpod.');
```

Comment:
Please add a trailing comma for consistency and readability.

Reply:
Done